config/locales/en.yml

```diff
@@ -96,6 +96,19 @@ en:
       title: I have a different problem
       text: For other problems with creating your account or signing in.
       link_text: contact us
+  check_email_password_reset:
+    heading: Check your email
+    body_one: We have sent you an email with instructions for how to reset your password.
```

Comment:
`body` could be an array that you iterate over

Reply:
You can also use html in `en.yml` so avoid the need for separate items per paragraph